packages/react-email/src/utils/get-email-component.ts

```diff
@@ -85,11 +85,18 @@ export const getEmailComponent = async (
     process,
   };
   const sourceMapToEmail = JSON.parse(sourceMapFile.text) as RawSourceMap;
+  // because it will have a path like <tsconfigLocation>/stdout/email.js.map
+  sourceMapToEmail.sourceRoot = path.resolve(sourceMapFile.path, '../..');
+  sourceMapToEmail.sources = sourceMapToEmail.sources.map((source) =>
+    path.resolve(sourceMapFile.path, '..', source),
+  );
   try {
     vm.runInNewContext(builtEmailCode, fakeContext, { filename: emailPath });
   } catch (exception) {
     const error = exception as Error;
 
+    error.stack &&= error.stack.split('at Script.runInContext (node:vm')[0];
```

Comment:
The error's stack here is always going to go back into Node's VM, but from this point onwards it's basically useless, so we split and only use what comes before.